monai/transforms/transforms.py

```diff
@@ -62,6 +62,24 @@ def __call__(self, img):
         return rescale_array(img, self.minv, self.maxv, self.dtype)
 
 
+@export
+class GaussianNoise:
+    """Add gaussian noise to image.
+
+    Args:
+        mean (float or array of floats): Mean or “centre” of the distribution.
+        scale (float): Standard deviation (spread) of distribution.
+        size (int or tuple of ints): Output shape. Default: None (single value is returned).
+    """
+
+    def __init__(self, mean=0.0, std=0.1):
+        self.mean = mean
+        self.std = std
+
+    def __call__(self, img):
+        return img + np.random.normal(self.mean, np.random.uniform(0, self.std), size=img.shape)
```

Comment:
To support the random logic in sync, I think you should extend **Randomizable** and use **self.R** instead of np.random.
Check the RandRotate90() example:
https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/transforms.py#L192
Thanks.